migration/mig_0013_add_rule_table.go

```diff
@@ -0,0 +1,83 @@
+/*
+Copyright © 2020 Red Hat, Inc.
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+    http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package migration
+
+/*
+   migration11 removes foreign keys to rules and rule error keys, and then removes the
+   rule and rule_error_key tables because it won't be stored in database anymore
+*/
+
+import (
+	"database/sql"
+
+	"github.com/RedHatInsights/insights-results-aggregator/types"
+)
+
+var mig0013AddRuleTable = Migration{
+	StepUp: func(tx *sql.Tx, driver types.DBDriver) error {
+		var err error
+		if driver == types.DBDriverPostgres {
+			_, err = tx.Exec(`
+				ALTER TABLE report DROP COLUMN report
+			`)
+		} else {
+			err = downgradeTable(
+				tx,
+				clusterReportTable,
+				`
+				CREATE TABLE report (
```

Comment:
It's gonna fuck up our database, you need to either clear the table completely(since we're caching the latest results, reports will fill the DB pretty quickly), or migrate rules to a new table

Reply:
Second is actually hard and I don't see a reason to do that